enterprise/utils.py

```diff
@@ -966,7 +967,7 @@ def traverse_pagination(response, endpoint):
     next_page = response.get('next')
     while next_page:
         querystring = parse_qs(urlparse(next_page).query, keep_blank_values=True)
-        response = endpoint.get(**querystring)
+        response = client.get(api_url, params=querystring).json()
```

Comment:
should error handling be added for response?

Reply:
done